target/targets.py

```diff
@@ -87,7 +87,7 @@ def reset_target(self, com_reset: Optional[bool] = False):
         """Resets the target. """
         self.target.reset_target()
         if com_reset and self.target_cfg.protocol == "ujson":
-            self._init_communication()
+            self.com_interface = self._init_communication()
```

Comment:
thanks. 
I was a bit puzzled but see that it was assigned in init only previousls. makes sense!